README.md

```diff
@@ -1,65 +1,62 @@
-# plastic-scm README
+# Plastic SCM integration with VS Code's SCM features
```

Comment:
What about adding our logo here at the top? I think it's already in master, so rebase your branch to add it if you want 😄

Reply:
I added the logo at the top in SVG format wrapped into an HTML <p>. This raised up a linter warning. In order to solve that, I added a rule to the markdown configuration file ("MD033": false).